test.rkt

```diff
@@ -78,4 +78,6 @@
   (-3     : (- 1 + * - 2))
   (1      : (- 1 + * - - 2))
   (-1/2   : (- 1 + * - / - 2))
+  (5      : (2 + 3 *))
+  (2/3    : (1 - 3 /))
```

Comment:
Do * and / have a defined postfix meaning here? Could we instead test with the example in your issue? I think that might be clearer.

Could we also test a postfix operator plus a prefix operator on the entire expression? For instance `(- 2 + 3 !)`.